src/ServerAdministration/WindowsService/WheelMUDService.cs

```diff
@@ -8,60 +8,37 @@
 // </summary>
 //-----------------------------------------------------------------------------
 
+using Topshelf;
```

Comment:
I looked at that link and not only is that reason weak, I would actually consider any code that needed this special placement to work to actually contain a defect due to it, and would consider the using inside the namespace to be a weak attempt to hide the problem/defect.  Its just plain sloppy in my opinion.

Types should be respected and the .Net run-time always wins; If you must extend a known type you can just add an extension method or rename your class/type so there is no conflicts. Now suddenly the need to put a using statement inside a namespace goes away.